src/components/browse/components/SortController.js

```diff
@@ -65,12 +65,17 @@ export class SortController extends React.PureComponent {
             'sortColumn'    : sortKey,
             'sortReverse'   : reverse
         };
-    });
+    }
 
     constructor(props){
         super(props);
         this.sortBy = this.sortBy.bind(this);
         this.state = { 'changingPage' : false }; // 'changingPage' = historical name, analogous of 'loading'
+
+        this.memoized = {
```

Comment:
Adjusted these to be memoized per-instance esp. since multiple instances of this component per view is now possible.